src/main/frontend/components/block.cljs

```diff
@@ -512,12 +512,13 @@
                                       :font-weight    500
                                       :max-height     600
                                       :padding-bottom 64}}
-                             (if (and (string? page-original-name) (string/includes? page-original-name "/"))
+                             (if (and (string? page-original-name) (text/namespace-page? page-original-name))
                                [:div.my-2
                                 (->>
-                                  (for [page (string/split page-original-name #"/")]
-                                    (when (and (string? page) page)
-                                      (page-reference false page {} nil)))
+                                  (for [namespace-page (gp-util/split-namespace-pages page-original-name)]
+                                    (when (and (string? namespace-page) namespace-page)
+                                      (let [label (second (gp-util/split-last "/" namespace-page))]
```

Comment:
We should avoid hardcoding ns chars where possible to control our uses of the ns feature. I can switch these to a var